utils/credentials.js

```diff
@@ -0,0 +1,12 @@
+const credentials = {
+	jira: {
+		url: '/MetricsAutomationReporter/Jira/Url',
+		basicAuth: '/MetricsAutomationReporter/Jira/BasicAuth',
+	},
+};
+
+const getCredentialsKeys = ({ toolName = 'jira' }) => credentials[toolName];
+
+module.exports = {
```

Comment:
any reason/pattern for us to not export the whole credentials object, and who consumes can get the specfic objects

like on jira.js we do a const { jira } = require('../utils/credentials');

and on this file we export credentials object

Reply:
No nothing in special @Henrique-Richter . We first created in this way because the initial idea was to have a function to model and export the data accordingly to the tool name. However, since we updated the way that the SSM receive the input, I believe that we can just export the data.